### DISCUSSION TITLE
Build Failed: Unexpected end of JSON input

### DISCUSSION THREAD

**Discussion by krljde** [vercel/community#1189] [Help] [answered]
### Summary

Hi everyone,

I'm having an issue with a build error when trying to deploy my project on Vercel. The error message is "Unexpected end of JSON input". I'm not sure which file is causing this issue and would love some help trying to figure it out. Has anyone else encountered this error before? Any suggestions on how to troubleshoot or resolve it would be greatly appreciated. Thank you in advance for your help!

### Additional information

![Build error](https://user-images.githubusercontent.com/105631151/209455115-a5146329-7693-41d7-9e58-743f8e430bc0.PNG)

**Comment by swarnava** [comment, marked as answer]
It is likely issue with your package.json. Please make sure it's properly configured. You can refer to https://jsonlint.com/ to validate your JSON.

**Reply by krljde** [reply]
Hey, thanks for your help! So it turns out I accidentally erased all the code inside of my project's package-lock.json file. I was able to fix it, after I updated my package-lock.json through npm install.
